json-rpc/src/util.rs

```diff
@@ -120,160 +110,6 @@ impl SdkInfo {
     }
 }
 
-pub fn vm_status_view_from_kept_vm_status(status: &KeptVMStatus) -> VMStatusView {
```

Comment:
it would be easier for every one if you could just submit a PR to refactor these logic (moving the logic around without changing anything).
in current format, I can't really review these change.

Reply:
@xli Philip does a really good job at breaking down PRs into individual atomic commits. Please use the "commit" view when reviewing these changes. It makes reviewing these changes much easier to read.

Reply:
Looks like github doesn't do a very good job of displaying the code move. FWIW, in that diff I didn't change anything except move the function bodies and comment out the move-explain part.

Reply:
right, I forgot to use commits....